sdk/schemaregistry/Azure.Data.SchemaRegistry/src/autorest.md

```diff
@@ -4,13 +4,14 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 title: SchemaRegistryClient
-require: https://github.com/Azure/azure-rest-api-specs/blob/70e53bf07d4f67000743c05d281930f2713a988e/specification/schemaregistry/data-plane/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/c364b64a6b412ffd7507dea71ae53251d35748c1/specification/schemaregistry/data-plane/readme.md
 generation1-convenience-client: true
 ```
 
 ## Swagger workarounds
 
 ### Update producers arrays - get version
+Because of some limitations of swagger and Autorest, Autorest cannot properly deduce that we would like to have an object return type to put on this method if we have a mix of "text/..." and "application/..." outputs in the produces array. Because of this we need to have "application/octet-stream" as a kind of stand-in for the the text outputs (Custom: "text/plain; charset=utf-8" and Protobuf: "text/vnd.ms.protobuf"). We have to determine what the output is on the client side based on the content-type.
```

Comment:
Added a comment because it took me a second to remember why we can only have the "application/octet-stream" in the produces array.